tests/compute/models/ssh_key_tests.rb

```diff
@@ -16,4 +16,26 @@
 
     returns(false) { @ssh_keys.get(ssh_key.href).nil? }
   end
+
+  tests('#create').succeeds do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.